Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1809,11 +1809,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
             maxHeight = '';
         }
 
-        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference);
+        const bufferFromLeft = 30;
+        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference) - bufferFromLeft;
         const minWidth = LayoutUtility.GetOuterWidth(reference);
         const maxWidth = Math.min (isNaN (options.maxWidth) ? LayoutUtility.GetOuterWidth(reference) : options.maxWidth, availableWidth);
-        const requiredWidth = contentWidth + 30;
-        const elementWidth = Math.max (Math.min (requiredWidth, maxWidth), minWidth);
+        const requiredWidth = contentWidth + 1; //to make sure that the inaccurate content width calculation is accurate enough
+        const maxAllowedWidth = Math.min(requiredWidth, maxWidth)
+        const elementWidth = Math.max(minWidth, maxAllowedWidth)
```

Comment:
If you want to split up the clamp here we need another name for `maxAllowedWidth` since that name does not fit the actual value.